explorer/nodewatch/nodewatch/__init__.py

```diff
@@ -91,6 +91,18 @@ def symbol_summary():  # pylint: disable=unused-variable
 		template_name, context = symbol_routes_facade.html_summary()
 		return render_template(template_name, **context)
 
+	def _get_json_nodes(role, exact_match, request_args):
+		only_ssl = request_args.get('only_ssl')
+
+		if only_ssl is not None:
+			only_ssl = only_ssl == '' or only_ssl.lower() == 'true'
```

Comment:
Should be able to rewrite 
``only_ssl = request_args.get('only_ssl', 'false').lower() in ('', 'true')``